tests/kafkatest/services/kafka/kafka.py

```diff
@@ -308,4 +347,4 @@ def bootstrap_servers(self):
 
         This is the format expected by many config files.
         """
-        return ','.join([node.account.hostname + ":9092" for node in self.nodes])
+        return ','.join([node.account.hostname + ":" + str(self.client_port) for node in self.nodes])
```

Comment:
One more thing per our offline conversation - maybe we can ditch `self.client_port` by specifying protocol as an optional argument to bootstrap_servers. E.g.
```
def bootstrap_servers(protocol="PLAINTEXT"):
    ...
```